### DISCUSSION TITLE
An installable shopping cart module

### DISCUSSION THREAD

**Discussion by eratech-de** [geodesicsolutions-community/geocore-community#106] [Ideas]
Please add one Shopping Cart Add-Ons. One of the biggest shortcomings in the software is this module.

**Comment by jonyo** [comment]
What do you mean by shopping cart module?  Like a full shopping cart running inside the software?  Have you seen the "buy now" features?

By the way, we're still in the process of getting community edition version 20 final released, with the goal of PHP 7.2 and PHP 8 compatibility.  Right now we're working on the 5th beta to that end.  I don't think any new features will make it in to version 20 before release.  But developers can always make an addon that works with the software!

We also welcome any developers that want to help out and contribute, and testers to help us find bugs!  😄
